src/mlpack/bindings/cli/in_place_copy.hpp

```diff
@@ -53,12 +53,34 @@ void InPlaceCopyInternal(
     const typename std::enable_if<
         arma::is_arma_type<T>::value ||
         std::is_same<T,
-                     std::tuple<mlpack::data::DatasetInfo, arma::mat>>::value ||
-        data::HasSerialize<T>::value>::type* = 0)
+                     std::tuple<mlpack::data::DatasetInfo, arma::mat>>::value>::type* = 0)
```

Comment:
```suggestion
                     std::tuple<mlpack::data::DatasetInfo, arma::mat>>::value
                 >::type* = 0)
```
Looks like this line was a little too long. :+1: